src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Represents a personal assistant chatbot to help a person keep track of various things.
+ */
+public class Duke {
+
+    /** UI for Duke to display messages. */
+    private final Ui ui;
+
+    /** Storage for Duke to store tasks. */
+    private final Storage storage;
+
+    /** Parser for Duke to parse inputs. */
+    private final Parser parser;
+
+    /**
+     * Creates an instance of Duke.
+     */
+    public Duke() {
+        TaskList taskList = new TaskList();
+        this.ui = new Ui();
+        this.storage = new Storage("../data", "../data/duke.txt", this.ui, taskList);
+        this.parser = new Parser(this.storage);
+    }
+
+    public static void main(String[] args) {
+        Duke duke = new Duke();
+        duke.run();
+    }
+
+    /**
+     * Runs Duke.
+     */
+    public void run() {
```

Comment:
Perhaps can consider changing the method name to become more descriptive. (e.g. runChatbot() or runApplication()).

Reply:
Changed to runDuke().